source/localizable/analytics/cps_privacy_policy.en.html.md.erb

```diff
@@ -0,0 +1,238 @@
+---
+parent: analytics
+title: cps_privacy_policy
+locale:
+  skroutz:
+    - 'el'
+    - 'en'
+---
+
+# Privacy Policy of CPC/CPS Mixed Billing Model
+
+<% I18n.with_locale('en') do %>
```

Comment:
It seems that the table of content (toc) plays well with localization without the need to explicitly set the locale. So let's ✂️ this

Reply:
I'm on it

Reply:
This is present on the Greek template as well, should I also remove it from there?